apps/client/src/pages/ErrorPage.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { useRouteError } from 'react-router-dom';
+
+const ErrorPage: React.FC = () => {
```

Comment:
Please add unit tests for the ErrorPage component.

Reply:
Thanks! Will test this.